OpenRA.Mods.Cnc/Traits/Buildings/ProductionAirdrop.cs

```diff
@@ -38,7 +37,7 @@ public ProductionAirdrop(ActorInitializer init, ProductionAirdropInfo info)
 			this.info = info;
 		}
 
-		public override bool Produce(Actor self, ActorInfo producee, string factionVariant)
+		public override bool Produce(Actor self, ActorInfo producee, string factionVariant, string exitType)
```

Comment:
Can you please work through the various files and change this from `exitType` to `productionType` to make it clearer what these are?